README.md

```diff
@@ -0,0 +1,230 @@
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+Modify this file to match your project and remove sections that don't apply.
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <!-- <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/> -->
+
+  <h3><b>React CAPSTONE README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    <!-- - [Tech Stack](#tech-stack) -->
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+
+  - [Setup](#setup)
+
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  <!-- - [Prerequisites](#prerequisites)
+  - [Deployment](#triangular_flag_on_post-deployment) -->
+
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Air Pollution] <a name="AirPollution"></a>
+
+> This is the project using reactjs.
+
+<!-- *[Bookstore]* can do math calculation like add, subtract, multiply and division. -->
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+> I use HTML/js  REACT for the front-end.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+<!-- > Describe between 1-3 key features of the application. -->
+
+- *[JS OBJECTS]*
+- *[BASIC HTML]*
+  <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+[LIVE DEMO](https://develop--zesty-mooncake-1a3672.netlify.app/)
+
+<!-- - [Live Demo Link]() -->
+
+
+
+https://user-images.githubusercontent.com/68339870/227716785-9e528307-5ecf-4663-a7ad-c574599736dd.mp4
+
+
+
+https://user-images.githubusercontent.com/68339870/227717467-2454ec53-0c92-492e-b0be-3d6121e44fb8.mp4
```

Comment:
- [ ] As required by the last reviewer, Kindly talk about a piece of the code you finds interesting or is proud of making.
and make that in one video.

- [ ] [Optional] It would be better if you upload this video to LOOM or Google Drive and only provide the link here in the README file.